src/Analyzers/CSharp/Tests/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -453,6 +454,52 @@ internal class Program
 {
     private volatile int x;
 }
+");
+        }
+
+        [WorkItem(48899, "https://github.com/dotnet/roslyn/issues/48899")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
+        public async Task TestAbstractMethod()
+        {
+            await VerifyCS.VerifyCodeFixAsync(@"
+public abstract class TestClass
+{
+    abstract string {|CS0621:[|Test|]|} { get; }
+}
+",
+@"
+public abstract class TestClass
+{
+    protected abstract string Test { get; }
+}
+");
+        }
+
+        [WorkItem(48899, "https://github.com/dotnet/roslyn/issues/48899")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
+        public async Task TestOverriddenMethod()
+        {
+            await VerifyCS.VerifyCodeFixAsync(@"
+public abstract class TestClass
+{
+    public abstract string Test { get; }
+}
+
+public class Derived : TestClass
+{
+    override string {|CS0507:{|CS0621:[|Test|]|}|} { get; }
+}
+",
+@"
+public abstract class TestClass
+{
+    public abstract string Test { get; }
+}
+
+public class Derived : TestClass
+{
+    public override string Test { get; }
+}
```

Comment:
could potentially add a test case for virtual members being public as well

Reply:
sure.  i'll add that in a followup :)